content/node-runners/setup/linux.md

```diff
@@ -33,7 +36,17 @@ In addition to downloading and installing our Node, these commands will also ins
 
 Once the installation is complete, check your service status.
 
-### Check service status
+### Latest non stable release 
+
+Latest non stable releases include bug fixes and improvements that have not yet been pushed
+for the whole user base and are not guaranteed to work.
```

Comment:
I think "non-stable" hint's at the fact that it's not yet - STABLE.
However I would point out that public should run non-stable releases at their own risk, no?

Reply:
I've seen some early access games naming such releases `I will back up my saves` releases :) maybe advice to back-up `/$USER/.mysterium` would avoid some issues? It's just a thought for discussion

Reply:
Both are good points. nice